spring-addons-starter-rest/src/test/java/com/c4_soft/springaddons/rest/reactive/SpringAddonsServerWebClientSupportAnonymousTest.java

```diff
@@ -0,0 +1,54 @@
+package com.c4_soft.springaddons.rest.reactive;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.matching.AbsentPattern;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.wiremock.spring.EnableWireMock;
+import org.wiremock.spring.InjectWireMock;
+import reactor.test.StepVerifier;
+
+
+@ActiveProfiles("forward-bearer")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@EnableWireMock
+class SpringAddonsServerWebClientSupportAnonymousTest {
```

Comment:
Maybe rename to `SpringAddonsServerWebClientSupportTest`: 
- more test cases could be added later
- the current test is actually asserting that the WebTestClient sends the request in case of missing security context (not of anonymous security context)